.pre-commit-hooks.yaml

```diff
@@ -77,6 +77,7 @@
   entry: hooks/golang/golangci-lint.sh
   language: script
   files: \.go$
+  pass_filenames: false
```

Comment:
@shakefu this is the key change, we might want to add this to some of the other checks too.

Reply:
Maybe with this change, the hook that I added below may no longer be needed, which is running for all the go modules in the repo and was probably trying to fix some of the same issues you run into

Reply:
If you look at https://github.com/turo/pre-commit-hooks/blob/main/hooks/golang/golangci-lint-mod.sh the end result is very similar so probably it woudl be good to consolidate into just one hook